src/control/control.h

```diff
@@ -30,6 +30,7 @@
 #endif
 
 #include "control/jobs.h"
+#include "control/crawler.h"
```

Comment:
Maybe not needed anymore?

Reply:
Apart this minor comment looks good. I'll test tomorrow. Thanks for the hard work with all the review and fixes/improvements!

Reply:
We use `dt_set_backthumb_time()` at quite a few places and would have to add that to multiple files ...

Reply:
Not blocking, but I feel that it is better to have the `#include` where routines are used.